specification/compute/resource-manager/Microsoft.Compute/GalleryRP/stable/2022-08-03/gallery.json

```diff
@@ -2424,6 +2429,9 @@
         "safetyProfile": {
           "$ref": "#/definitions/GalleryImageVersionSafetyProfile"
         },
+        "securityProfile": {
```

Comment:
Move this property down 1 so it is at the end of the property list and regenerate the .net sdk for that.